releasenotes/notes/tornado-none-bug-85faed75c859f5e5.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Handle when the current span is None in log_exception().
```

Comment:
hindsight, should this say "tornado" somewhere?

Reply:
Ah shoot I think you're right.